README.md

```diff
@@ -1,65 +1,86 @@
+   <tr>
+     <td><strong>Input widgets</strong><br><br><img src="https://docs.streamlit.io/images/api/date_input.jpg" height="150"></td>
+     <td><strong>Celebrations<br><br><img src="https://docs.streamlit.io/images/api/balloons.jpg" height="150"></td>
+     <td><strong>Multi-page apps<br><br><img src="https://docs.streamlit.io/images/mpa-add-pages.png" height="150"></td>
+     <td><strong>Charts<br><br><img src="https://docs.streamlit.io/images/api/line_chart.jpg" height="150"></td>
+     <td><strong>Layout<br><br><img src="https://docs.streamlit.io/images/api/tabs.jpg" height="150"></td>
+     <td><strong>Media<br><br><img src="https://docs.streamlit.io/images/api/image.jpg" height="150"></td>
+  </tr>
+<table>
 
-The complete demo is implemented in less than 300 lines of Python. In fact, the app contains [only 23 Streamlit calls](https://github.com/streamlit/demo-self-driving/blob/master/streamlit_app.py) which illustrates all the major building blocks of Streamlit. You can try it right now at [share.streamlit.io/streamlit/demo-self-driving](https://share.streamlit.io/streamlit/demo-self-driving).
+Our vibrant creators community also extends Streamlit capabilities using [Streamlit Components](https://streamlit.io/components) 🧩
 
-## The Streamlit GitHub badge
+## Get inspired 
+ 
+There's so much you can build with Streamlit!
+- 🧬  [Science & technology apps](https://streamlit.io/gallery?category=science-technology)
+- 💬  [NLP & language apps](https://streamlit.io/gallery?category=nlp-language)
+- 👀  [Computer vision & images](https://streamlit.io/gallery?category=computer-vision-images)
+- 🏦  [Finance & business](https://streamlit.io/gallery?category=finance-business)
+- 🗺  [Geography & society](https://streamlit.io/gallery?category=geography-society)
+- ...
 
-Streamlit's GitHub badge helps others find and play with your Streamlit app.
+**Discover more in [our gallery](https://streamlit.io/gallery)** 🎈
 
-[![Streamlit App](https://static.streamlit.io/badges/streamlit_badge_black_white.svg)](https://share.streamlit.io/streamlit/demo-face-gan)
+## Community Cloud
+  
+Deploy, manage and share your apps for free using our [Community Cloud](https://streamlit.io/cloud)! Sign-up [here](https://share.streamlit.io/signup). <br><br>
+<img src="https://user-images.githubusercontent.com/7164864/214965336-64500db3-0d79-4a20-8052-2dda883902d2.gif" width="400"></img>
 
-Once you deploy your app, you can embed this badge right into your GitHub readme.md as follows:
+## Other
 
+- Learn all about Streamlit in [our docs](https://docs.streamlit.io)
+- Ask/answer questions and showcase your work in [our forum](https://discuss.streamlit.io)
+- Stay up to date on our latest news and releases in [our blog](https://blog.streamlit.io)
+- Extend Streamlit capabilities by installing or creating your own [Streamlit Components](https://streamlit.io/components) 🧩
+- Help others find and play with your app by using the Streamlit GitHub badge in your repository
```

Comment:
These bullet points are great but I feel like they could be a bit shorter. It's the end of the readme, attention span is dropping ;)